e2e/charts/e2e-resources/Chart.yaml

```diff
@@ -1,10 +0,0 @@
-apiVersion: v2
-version: 1.2.0
-appVersion: "1.0.0"
-description: This chart creates e2e resources.
-name: e2e-resources
-
-dependencies:
-  - name: kube-state-metrics
-    version: 2.13.2
-    repository: https://kubernetes.github.io/kube-state-metrics
```

Comment:
Why is this deleted? Is this not going to make this chart uninstallable?

Reply:
Ah. It has so many changes that git thinks is a new file: https://github.com/newrelic/nri-kubernetes/pull/377/files#diff-a8dbaa85f24d99fa37816cd13be16708b3eab7f9b9fd58fb2908d1ff073f5b6aR1-R24

Ok.